### PR TITLE
Added a list of minutes

### PR DESCRIPTION
**Pull request by invliD**

### CODE REVIEW

File: _1327/minutes/views.py [hunk 1]
```diff
@@ -16,6 +17,24 @@
 from .models import MinutesDocument
 
 
+@login_required
+def list(request):
+	years = {}
+	minutes = MinutesDocument.objects.order_by('-date')
+	for m in minutes:
```
Comment (Bartzi) [inline _1327/minutes/views.py]: could you name it `minute` instead of `m`?

Comment (invliD) [inline _1327/minutes/views.py]: You don't use the singular. Minutes are always plural.

Comment (Bartzi) [inline _1327/minutes/views.py]: oh okay in that case...

**Comment by Bartzi** [comment]
:+1:

**Pull request merged**